src/scripts/wildBattle/Routes.ts

```diff
@@ -424,7 +423,15 @@ Routes.add(new RegionRoute(
         water: ['Magikarp', 'Poliwag'],
         headbutt: ['Venonat', 'Exeggcute', 'Hoothoot', 'Pineco'],
     }),
-    [new RouteKillRequirement(10, GameConstants.Region.johto, 42)]
+    [
+        new OneFromManyRequirement([
+            new MultiRequirement([
+                new ClearDungeonRequirement(1, GameConstants.getDungeonIndex('Mt Mortar')),
+                new GymBadgeRequirement(BadgeEnums.Fog),
+            ]),
+            new RouteKillRequirement(10, GameConstants.Region.johto, 42)
```

Comment:
build fail, should this be `new RouteKillRequirement(10, GameConstants.Region.johto, 42),` instead (comma at the end)